proco/connection_statistics/serializers.py

```diff
@@ -39,3 +45,34 @@ class Meta:
             'connectivity_availability',
         )
         read_only_fields = fields
+
+
+class RealTimeConnectivitySerializer(serializers.ModelSerializer):
+    school = SeparatedReadWriteField(write_field=serializers.IntegerField(),
```

Comment:
no need to use write_field here. PKRelatedField will be generated automatically if not provided